packages/cli/lib/lib/webpack/webpack-client-config.js

```diff
@@ -109,6 +109,10 @@ async function clientConfig(env) {
 						from: resolve(__dirname, '../../resources/sw-debug.js'),
 						to: 'sw-debug.js',
 					},
+					existsSync(source('static/root')) && {
+						from: resolve(source('static/root')),
+						to: resolve(cwd),
```

Comment:
I think this puts them in the repo root. I believe should be `to: '.'` to place them in `output.path`.